sw/airborne/mcu_periph/usb_serial.h

```diff
@@ -32,7 +32,16 @@
 #include "std.h"
 #include "pprzlink/pprzlink_device.h"
 
+// we really need just 1 byte
+#define USB_RX_BUFFER_SIZE 10
+
 struct usb_serial_periph {
+  /** Receive buffer */
+  uint8_t rx_buf[USB_RX_BUFFER_SIZE];
```

Comment:
Maybe make this the same size as the UART buffers?

Reply:
I'm not found of including uart header here, but why not. At least USB buffer should be configurable independently of uart buffer.

Reply:
Its configurable now. But its really just an buffer around the real buffer to be compliant with the `char_available`, `get_char` interface.